addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -358,15 +371,13 @@ const Wysiwyg = Widget.extend({
             this._setONotEditable(this.odooEditor.editable);
         });
 
-        return _super.apply(this, arguments).then(() => {
-            if (this.options.autohideToolbar) {
-                if (this.odooEditor.isMobile) {
-                    $(this.odooEditor.editable).before(this.toolbar.$el);
-                } else {
-                    $(document.body).append(this.toolbar.$el);
-                }
+        if (this.options.autohideToolbar) {
+            if (this.odooEditor.isMobile) {
+                $(this.odooEditor.editable).before(this.toolbar.$el);
+            } else {
+                $(document.body).append(this.toolbar.$el);
             }
-        });
+        }
```

Comment:
This used to return the call to `super` and doesn't do it anymore. Was this intended ?